lib/mongo.js

```diff
@@ -50,6 +50,13 @@ var convert = function(query) {
             queryAll[group.field][mType] = _.flatten(queryAll[group.field][mType]);
         }
 
+        // simplify "!= true" to "= false"
+        else if (group.type === "!=" && group.value === true) {
+          group.type = "=";
+          group.value = false;
+          appendGroup(group, _.constant(group.value));
+        }
+
```

Comment:
The idea here is to simplify the eventual query. Instead of something like:
```js
{ "field": { "$ne": true } }
```
we would get:
```js
{ "field": false }
```
which, IMHO, is more elegant and readable.

While my current use-case is the conversion of `!= true` to `= false` when using `NOT is:field`, I definitely see the possible need to convert `!= false` to `= true`.

Adding this is trivial, so I may just push another commit to add.